pkg/certificate/manager.go

```diff
@@ -88,88 +84,31 @@ func NewManager(
 		webhookType:   webhookType,
 		now:           time.Now,
 		certsDuration: certsDuration,
-		log: logf.Log.WithName("webhook/server/certificate/manager").
+		log: logf.Log.WithName("certificate/manager").
 			WithValues("webhookType", webhookType, "webhookName", webhookName),
 	}
 	return m
 }
 
-// Start the cert manager until stopCh is closed, the cert manager is in charge
-// rotation of certificates if needed.
-//
-// It  implemenets Runnable [1] so manager can add this to a
-// controller runtime manager
-//
-// [1] https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/manager/manager.go#L208
-func (m *Manager) Start(stopCh <-chan struct{}) error {
-	m.log.Info("Starting cert manager")
-
-	m.loadCACertFromCABundle()
-
-	wait.Until(func() {
-		m.waitForDeadlineAndRotate()
-	}, time.Second, stopCh)
-
-	return nil
-}
-
-// In case the manager is restarted we have to load the current CA instead
-// of generating new one
-func (m *Manager) loadCACertFromCABundle() {
+func (m *Manager) firstCACertFromCABundle() (*x509.Certificate, error) {
```

Comment:
Yep, good suggestiong, I will use `get` since it's ready doing a kubectl get kind of thing, usually at golang `accessors` don't have the `get` prefix, but this is more than getting a field at memory so we good with `get`.